### DISCUSSION TITLE
footnote doesn't work

### DISCUSSION THREAD

**Discussion by ggous** [quarto-dev/quarto-cli#1220] [Q&A] [answered]
Hello, 
I am  trying to use a footnote.
I have created a quarto project for writting a book.

I tried to use:

```
^[this is a footnote]

[^this is a footnote]

here is footnote ref [^1]

[^1]:this is a footnote
```

but nothing from the above works!

**Comment by dragonstyle** [comment]
Can you provide an example that shows this behavior? I took this example from the docs and it worked fine:

```
Here is a footnote reference,[^1] and another.[^longnote]

[^1]: Here is the footnote.

[^longnote]: Here's one with multiple blocks.

    Subsequent paragraphs are indented to show that they
belong to the previous footnote.

        { some.code }

    The whole paragraph can be indented, or just the first
    line.  In this way, multi-paragraph footnotes work like
    multi-paragraph list items.

This paragraph won't be part of the note, because it
isn't indented.
```

![Screen Shot 2022-06-24 at 9 10 36 AM](https://user-images.githubusercontent.com/261654/175542781-7e83bd8d-c80c-4880-bcf3-52b302539649.png)

**Comment by ggous** [comment, marked as answer]
Hmm.. 

I am running `Rstudio` and I was using the `Visual` representation instead of the `Source` !
 
That was the problem!
